src/cli.ts

```diff
@@ -105,6 +106,7 @@ async function main(argv: string[]) {
   const unframeGrpcWebJsonRequestsHostnames: string[] =
     options.unframeGrpcWebJsonRequestsHostname;
   const rewriteBeforeDiffRules: RewriteRules = options.rewriteBeforeDiff;
+  const match: boolean = options.match === undefined ? false : options.match;
```

Comment:
**Suggestion**: Rename `match` to `exactRequestMatching` throughout this changeset to make the intent of the flag more self-descriptive.

Reply:
Done